packages/m2m-gateway/test/integration/tenants/getCertifiedAttributes.test.ts

```diff
@@ -0,0 +1,187 @@
+    id: mockApiAttribute1.data.id,
+    assignmentTimestamp: new Date().toISOString(),
+  };
+
+  const mockTenantAttribute2: tenantApi.CertifiedTenantAttribute = {
+    id: mockApiAttribute2.data.id,
+    assignmentTimestamp: new Date().toISOString(),
+  };
+
+  const mockApiTenant: tenantApi.Tenant = {
+    ...getMockedApiTenant().data,
+    attributes: [
+      { certified: mockTenantAttribute1 },
+      { certified: mockTenantAttribute2 },
+    ],
+  };
+
+  const mockGetTenantResponse: WithMaybeMetadata<tenantApi.Tenant> = {
+    data: mockApiTenant,
+    metadata: undefined,
+  };
+  const mockGetBulkedAttributesResponse: WithMaybeMetadata<attributeRegistryApi.Attributes> =
+    {
+      data: {
+        results: [mockApiAttribute1.data, mockApiAttribute2.data],
+        totalCount: 2,
+      },
+      metadata: undefined,
+    };
+
+  const mockGetTenant = vi.fn().mockResolvedValue(mockGetTenantResponse);
+  const mockGetBulkedAttributes = vi
+    .fn()
+    .mockResolvedValue(mockGetBulkedAttributesResponse);
+
+  mockInteropBeClients.tenantProcessClient = {
+    tenant: {
+      getTenant: mockGetTenant,
+    },
+  } as unknown as PagoPAInteropBeClients["tenantProcessClient"];
+
+  mockInteropBeClients.attributeProcessClient = {
+    getBulkedAttributes: mockGetBulkedAttributes,
+  } as unknown as PagoPAInteropBeClients["attributeProcessClient"];
+
+  beforeEach(() => {
+    // Clear mock counters and call information before each test
+    mockGetTenant.mockClear();
+    mockGetBulkedAttributes.mockClear();
+  });
+
+  it("Should succeed and perform API clients calls", async () => {
+    const m2mCertifiedAttributeResponse1: m2mGatewayApi.TenantCertifiedAttribute =
+      {
+        id: mockApiAttribute1.data.id,
+        description: mockApiAttribute1.data.description,
+        name: mockApiAttribute1.data.name,
+        code: mockApiAttribute1.data.code as string,
+        origin: mockApiAttribute1.data.origin as string,
+        assignedAt: mockTenantAttribute1.assignmentTimestamp,
+      };
+
+    const m2mCertifiedAttributeResponse2: m2mGatewayApi.TenantCertifiedAttribute =
+      {
+        id: mockApiAttribute2.data.id,
+        description: mockApiAttribute2.data.description,
+        name: mockApiAttribute2.data.name,
+        code: mockApiAttribute2.data.code as string,
+        origin: mockApiAttribute2.data.origin as string,
+        assignedAt: mockTenantAttribute2.assignmentTimestamp,
+      };
+
+    const m2mTenantsResponse: m2mGatewayApi.TenantCertifiedAttributes = {
+      pagination: {
+        limit: mockParams.limit,
+        offset: mockParams.offset,
+        totalCount: 2,
+      },
+      results: [m2mCertifiedAttributeResponse1, m2mCertifiedAttributeResponse2],
+    };
+
+    const result = await tenantService.getCertifiedAttributes(
+      unsafeBrandId(mockApiTenant.id),
+      mockParams,
+      getMockM2MAdminAppContext()
+    );
+
+    expect(result).toEqual(m2mTenantsResponse);
+  });
```

Comment:
Add check that the clients API calls were called with the right params with `expectApiClientGetToHaveBeenCalledWith`